kong/plugin_service_test.go

```diff
@@ -107,6 +107,36 @@ func TestPluginWithTags(T *testing.T) {
 	assert.NoError(err)
 }
 
+func TestPluginWithOrdering(T *testing.T) {
+	RunWhenEnterprise(T, ">=3.0.0", RequiredFeatures{})
+	assert := assert.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
+
+	plugin := &Plugin{
+		Name: String("request-termination"),
+		Ordering: &PluginOrdering{
+			Before: PluginOrderingPhase{
+				"access": []string{"key-auth", "basic-auth"},
```

Comment:
What do you think about testing that:
- adding a not-existing plugin in `PluginOrderingPhase` errors out
- adding a random key in `PluginOrderingPhase` errors out

Reply:
9723ed58466bfc917b0f3909b39347812621a736